projects/applicationinsights-angularplugin-js/src/lib/applicationinsights-angularplugin-error.service.ts

```diff
@@ -37,6 +37,8 @@ export class ApplicationinsightsAngularpluginErrorService implements IErrorServi
     }
 
     handleError(error: any): void {
+        console.error(error);
```

Comment:
We don't want to do this!

If you want to "see" the exception and your running in a browser (for development) then I would suggest that you use the Chrome Debug Extension (which is a Chromium extension) https://github.com/microsoft/ApplicationInsights-JS/tree/master/tools/chrome-debug-extension or you could use the Debug Plugin https://github.com/microsoft/ApplicationInsights-JS/tree/master/extensions/applicationinsights-debugplugin-js

Reply:
Another option would be to use the Diagnostic Logger `errorToConsole` and include it behind a configuration so that only if someone wants this enabled would they use it

```
if (someEnabledConfig && this.analyticsPlugin) {
    this.analyticsPlugin.logger.errorToConsole("Error: " + dumpObj(error));
}
```
The dumpObj helper from the AI core formats the error to include some more information rather than just the message.